### ISSUE TITLE
Registration form with User element integration can't send Email Verify activation email

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#686]
**Description**

I'm using a Formie form for user registration, using an Element Integration with the User Integration Provider. In my form's integration settings, __Activate User__ is off, and __Send Activation Email__ is on.

The activation email that's sent by Craft is always the "Password Reset" activation email, instead of the "Email Verify" one. I have a Password field in my registration form that is mapped to the User's password field in the integration settings.

It appears that the reason this is happening is because the Element integration is sending the activation email on these lines:
https://github.com/verbb/formie/blob/1e3fdfb364d83192df6d56f0102588de14ea6891/src/integrations/elements/User.php#L269-L277

But this is before the `$user` object actually has a password, which (as far as I understand it) isn't saved until here:
https://github.com/verbb/formie/blob/1e3fdfb364d83192df6d56f0102588de14ea6891/src/integrations/elements/User.php#L301-L303

The problem is that Craft's `Users::sendActivationEmail()` method that's being called on line 275 checks whether the `$user` object already has a password, and only sends the "Email Verify" email if it does. If it doesn't, Craft sends the "Password Reset" email instead:
https://github.com/craftcms/cms/blob/031be82d64346dc7af17fab0512fb983413bb411/src/services/Users.php#L339-L352

It seems that the result of this is that Formie can never cause the "Email Verify" activation email to be sent; only the "Password Reset" one.

Would it be possible to make it so the Email Verify activation email is sent if the registration form contains a password field that's mapped to the User's password attribute?

**Additional info**
- Plugin version: 1.5.1
- Craft version: 3.7.25.1
- Multi-site: No

**Comment by engram-design** [comment]
Ah, good point. The password for users is a bit tricky, because the User model only accepts a `newPassword` param, which is the plain text version of the password. The Password field already hashes it, so we can't use that, and it's impossible to save a hashed password on a User element. That's the reason for the direct `Db::update()` call.

But well picked up about the `sendActivationEmail()` on Craft's end. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
@engram-design Thanks so much for that; it works perfectly now!

**Comment by engram-design** [comment]
Fixed in [1.5.2](https://github.com/verbb/formie/releases/tag/1.5.2)
